versioned_docs/version-6.x/drawer-layout.md

```diff
@@ -38,7 +38,7 @@ Then, you need to install and configure the libraries that are required by the d
    npm install react-native-gesture-handler react-native-reanimated
    ```
 
-   The Drawer supports both Reanimated 1 and Reanimated 2. If you want to use Reanimated 2, make sure to configure it following the [installation guide](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation).
+   The Drawer supports the latest version of Reanimated. Make sure to configure it following the [installation guide](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/getting-started).
```

Comment:
This isn't quite correct. It supports both Reanimated 1 and the latest version. Reanimated 1 has a different API and supported via `useLegacyImplementation` prop. The latest version of Reanimated needs additional configuration.

Basically what was written before, but Reanimated 2 should be replaced by latest version.

Reply:
@satya164 You're right, sorry for the confusion. Sent a fix, let me know if it's good now!